src/Messages/SlackAttachment.php

```diff
@@ -341,14 +341,14 @@ public function footerIcon($icon)
     }
 
     /**
-     * Set the timestamp.
+     * Set the timestamp by passing a DateTime or a DateInterval instance, or a given number of seconds to delay.
      *
-     * @param  \DateTimeInterface|\DateInterval|int  $timestamp
+     * @param  \DateTimeInterface|\DateInterval|int  $delay
      * @return $this
      */
-    public function timestamp($timestamp)
+    public function timestamp($delay)
```

Comment:
Would you consider this a breaking change due to Named Arguments in PHP 8? https://wiki.php.net/rfc/named_params

Reply:
@nahanheffley not in this case because there's only one argument so there's no need to use named arguments here.

Reply:
Even though it's a bad idea to use named parameters for a function with one required parameter) somebody who is calling `timestamp(timestamp: 12345)` will suddenly start getting an error if they update this package. Along with @taylorotwell's comment it seems like changing the name is probably not the best idea.

I think a comment explaining that an integer value will be used as a delay, and then fix the issue in the next major release to not use delays at all.

Reply:
We have never said our BC policy supports named parameters. We're not adopting that policy right now.

Reply:
Thanks @taylorotwell, that is really what I meant to ask in my first question 👍